HW2/Non-Linear/cmd.ap.sub_01

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env tcsh
```

Comment:
Looks like this is for the motor localizer and should not be part of the homework.